Project.toml

```diff
@@ -14,13 +14,16 @@ PaddedViews = "5432bcbf-9aad-5242-b902-cca2824c8663"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
-ColorTypes = ">= 0.8.0"
-FixedPointNumbers = ">= 0.6.1"
-MappedArrays = ">= 0.2"
-OffsetArrays = ">= 0.8"
-PaddedViews = ">= 0.4.1"
-Reexport = ">= 0.2"
-julia = ">= 1.0"
+Colors = "0.9"
+ColorTypes = "0.8"
+FixedPointNumbers = "0.6.1"
```

Comment:
Is this specific patch release required?

Reply:
You'll note the old one was `>= 0.6.1`. [IIUC](https://julialang.github.io/Pkg.jl/v1/compatibility/), this allows anything in the range `[0.6.1, 0.7)`.

Reply:
I interpreted the documentation to mean that in order to have the range `[0.6.1, 0.7)` you need to specify the caret, e.g. `^0.6.1`.

The documentation you linked to gives this example:
`PkgD = "^0.2.3" # [0.2.3, 0.3.0)`

Reply:
Caret specifiers are the default though so I believe @timholy's correct here.

Reply:
FYI, this is also documented in [BlueStyle](https://github.com/invenia/BlueStyle#package-version-specifications) as a recommendation.